test/integration/util/util.go

```diff
@@ -91,27 +107,21 @@ func (m *MinishiftRunner) SetEnvFromEnvCmdOutput(dockerEnvVars string) error {
 }
 
 func (m *MinishiftRunner) GetStatus() string {
-	return strings.Trim(m.RunCommand("status", true), " \n")
+	cmdOut, _, _ := m.RunCommand("status")
+	return strings.Trim(cmdOut, " \n")
 }
 
-func (m *MinishiftRunner) CheckStatus(desired string) {
-	s := m.GetStatus()
-	if s != desired {
-		m.T.Fatalf("Machine is in the wrong state: %s, expected  %s", s, desired)
-	}
+func (m *MinishiftRunner) CheckStatus(desired string) bool {
+	return strings.Contains(m.GetStatus(), desired)
 }
 
 type OcRunner struct {
-	T          *testing.T
-	BinaryPath string
+	CommandPath string
 }
 
-func NewOcRunner(t *testing.T) *OcRunner {
-	p, err := exec.LookPath("kubectl")
-	if err != nil {
-		t.Fatal("Couldn't find kubectl on path.")
-	}
-	return &OcRunner{BinaryPath: p, T: t}
+func NewOcRunner() *OcRunner {
+	p, _ := exec.LookPath("kubectl")
```

Comment:
It is unused code for now... I changed it anyways as I saw another small change I could make.